Formula/visp.rb

```diff
@@ -0,0 +1,65 @@
+class Visp < Formula
+  desc "Visual Servoing Platform library"
+  homepage "https://visp.inria.fr"
```

Comment:
Should have a terminating slash `https://visp.inria.fr/`

Reply:
ok